bigquery_etl/cli/alchemer.py

```diff
@@ -6,7 +6,22 @@
 from bigquery_etl.alchemer.survey import get_survey_data, insert_to_bq
 
 
-@click.group(help="Commands for importing alchemer data.")
+@click.group(
+    help="""Commands for importing alchemer data.
+
+Examples:
+
+\b
```

Comment:
The `\b` marker indicates that the formatting is to be preserved for the following block

Reply:
At what level is the formatting getting stripped? Can we use markdown here? It would be good to avoid custom formatting concepts if possible, so any context you can provide about how you arrived at the `\b` solution would be appreciated.

Reply:
This is just to ensure that the formatting in the CLI looks right. Without these markers click puts a paragraph on a single line, which looks like this:

```
./bqetl query backfill --help
Usage: bqetl query backfill [OPTIONS] NAME

  Run a backfill for a query. Additional parameters will get passed to bq.

  Examples:

  # Backfill for specific date range # second comment line ./bqetl query
  backfill telemetry_derived.ssl_ratios_v1 \   --start_date=2021-03-01 \
  --end_date=2021-03-31

  # Dryrun backfill for specific date range and exclude date ./bqetl query
  backfill telemetry_derived.ssl_ratios_v1 \   --start_date=2021-03-01 \
  --end_date=2021-03-31 \   --exclude=2021-03-03 \   --dry_run
```

Click automatically wraps long lines, but the parameters and multiline comments do not look right.

Reply:
I hadn't understood that this was a Click feature that's documented: https://click.palletsprojects.com/en/7.x/documentation/#preventing-rewrapping

I thought we had custom logic here to interpret `\b`. I feel better about this now.

Reply:
Right, I should have linked to the docs 👍